src/main/webapp/user-page.html

```diff
@@ -31,11 +31,15 @@
     </nav>
     <h1 id="page-title">User Page</h1>
 
-    <form id="message-form" action="/messages" method="POST" class="hidden">
+    <form id="message-form" action="/messages" method="POST" enctype="multipart/form-data" class="hidden">
       Enter a new message:
       <br/>
       <textarea name="text" id="message-input"></textarea>
       <br/>
+      Add an image to your message:
```

Comment:
just a user experience suggestion/extensions - would this be equivalent to say, texting an image to a friend? If so, perhaps an extension of the current front end would be regex matching on a url, so if it detects an image url, it can immediately process it as sending an image. just an idea, but currently this seems the most intuitive, especially for our open project purposes!

Reply:
Right now, there's both image parsing using regex and the ability to upload images. Do you have a suggestion on what to name uploading images or keep it as is? It would be equivalent to texting an image to a friend.

Reply:
I think for our purposes right now, it works fine! Just a possible extension in the future that we could maybe keep in mind!